### ISSUE TITLE
inconsistent hover styling on menu items

### ISSUE THREAD

**Issue by adamgall** [decentdao/decent-interface#1889]
<img width="284" alt="Screenshot 2024-05-22 at 11 53 50 AM" src="https://github.com/decentdao/decent-interface/assets/62394931/010071e4-3f8c-4463-8eb2-c6a172fc76fd">
<img width="199" alt="Screenshot 2024-05-22 at 11 53 59 AM" src="https://github.com/decentdao/decent-interface/assets/62394931/4335f004-71fb-4201-927f-de71c3ee015d">
<img width="219" alt="Screenshot 2024-05-22 at 11 53 40 AM" src="https://github.com/decentdao/decent-interface/assets/62394931/f19d33dc-9161-42a5-9e37-53a658270d77">
@decentdao/engineering inconsistent hover styling on menu items. Let's match all of those to the sidebar behavior.

_Originally posted by @nicolaus-sherrill in https://github.com/decentdao/decent-interface/discussions/1863#discussioncomment-9524718_

**Comment by DarksightKellar** [comment]
@nicolaus-sherrill Could you please point me to up-to-date designs if any, as what I've found (and I'm sure what was worked with) on Figma are also fairly inconsistent:

![Image](https://github.com/decentdao/decent-interface/assets/7101382/e54e067d-88d5-4bb9-bf95-3224ce366f28)
![Image](https://github.com/decentdao/decent-interface/assets/7101382/31234226-c4ed-4c53-bc59-c42efa85547c)
![Image](https://github.com/decentdao/decent-interface/assets/7101382/8a9aba7c-8f02-4887-80be-cc02c8ee027d)

Basically, when do we want dividers? Do we _never_ want dividers? What should the hover padding look like? Etc. I know you've mentioned "match all of those to the sidebar behavior," but I gotta understand the intent before making blanket changes and risking having to circle back to this because some intention got swallowed up by the changes.

**Comment by xraystyle1980** [comment]
@DarksightKellar updated in Figma with consistent hover patterns and padding that match the sidebar. (4px padding on all sides, 4px border radius)

Dividers are intentional and needed between each menu item. 
[Updated in Figma](https://www.figma.com/design/ur5hElxplkmlY9gnGKlwrm/Components-%26-Styles---Decent-Design-System-1.0.0?m=auto&node-id=2168-20014&t=b3gctUAUxuhA2RlX-1)

![Image](https://github.com/decentdao/decent-interface/assets/18615633/1b992f11-4837-4db3-85ac-3ca25a26b33a)
